source/lac/sparsity_tools.cc

```diff
@@ -326,6 +331,66 @@ namespace SparsityTools
   }
 
 
+  void color_sparsity_pattern (const SparsityPattern     &sparsity_pattern,
+                               std::vector<int> &color_indices)
+  {
+    // Make sure that ZOLTAN is actually
+    // installed and detected
+#ifndef DEAL_II_TRILINOS_WITH_ZOLTAN
+    (void)sparsity_pattern;
+    AssertThrow (false, ExcZOLTANNotInstalled());
+#else
+    //Zoltan coloring algorithm is run in series by each processor. All objects colored are assumed to belong to Processor 0.
```

Comment:
> All objects colored are assumed to belong to Processor 0.

i don't think that's the case, you assume that exactly the same objects are there on each MPI process.

Reply:
Here and below, can you break long lines at approximately 80 characters?